docs/asciidoc/exportCypher.adoc

```diff
@@ -176,3 +192,47 @@ CREATE CONSTRAINT ON (node:`Bar`) ASSERT node.`name` IS UNIQUE;
 commit
 schema await
 ----
+
+==== New method:
+
+With the optimizations
+
+[source,cypher]
+----
+CALL apoc.export.cypher.all({fileName},{config})
+----
+Result:
+[source,cypher]
+----
+BEGIN
+CREATE INDEX ON :`Bar`(`first_name`,`last_name`);
+CREATE INDEX ON :`Foo`(`name`);
+CREATE CONSTRAINT ON (node:`Bar`) ASSERT node.`name` IS UNIQUE;
+CREATE CONSTRAINT ON (node:`UNIQUE IMPORT LABEL`) ASSERT node.`UNIQUE IMPORT ID` IS UNIQUE;
+COMMIT
+SCHEMA AWAIT
+BEGIN
+UNWIND [{`UNIQUE IMPORT ID`: 0, properties: {`born`:date('2018-10-31'), `name`:"foo"}}, {`UNIQUE IMPORT ID`: 4, properties: {`born`:date('2017-09-29'), `name`:"foo2"}}] as row
```

Comment:
we could use a shorter name here than  UNIQUE IMPORT ID so it's less to parse.
we should only backtick-quote fields that need quoting.

Reply:
remember, make it easier/faster for the parser :)

Reply:
We reuse (where possibile) the same structures of the `CREATE` export is it good for you if I change the shared names (`UNIQUE IMPORT ID` and `UNIQUE IMPORT LABEL`)?

Reply:
I meant in the params, not for the nodes.

Reply:
ok

Reply:
please update the docs too

Reply:
also with the non-multi-label on MATCH/MERGE